internal/pkg/plugin/helminstaller/helminstaller.go

```diff
@@ -0,0 +1,73 @@
+package helminstaller
+
+import (
+	"os"
+	"strings"
+
+	"github.com/devstream-io/devstream/internal/pkg/configmanager"
+	"github.com/devstream-io/devstream/internal/pkg/plugin/helminstaller/defaults"
+	"github.com/devstream-io/devstream/internal/pkg/plugininstaller/helm"
+	"github.com/devstream-io/devstream/pkg/util/log"
+	"github.com/devstream-io/devstream/pkg/util/types"
+)
+
+func GetDefaultOptionsByInstanceID(instanceID string) *helm.Options {
+	for name, options := range defaults.DefaultOptionsMap {
+		if strings.Contains(instanceID, name) {
```

Comment:
You should consider the situation that more than one tools have same prefix, such as `argocd` and `argocdapp`.

I have two solutions:

1. `strings.Contains(instanceID, name+"-")`
2. Calculate the match length of each tool and select the longest

Reply:
"name-" is a good idea. btw, that's the helm-installer plugin, argocdapp won't be here. We haven't support 2 tools with same prefix. If it will in the future, the problem could be resolve at that time. But I still agree with you, the "-" is a good idea.